package.json

```diff
@@ -42,6 +42,7 @@
     "react-router-dom": "^5.0.0",
     "react-scripts": "2.1.8",
     "redux": "^4.0.1",
+    "redux-devtools-extension": "^2.13.8",
```

Comment:
Personally I prefer more to use https://github.com/zalmoxisus/redux-devtools-extension

Reply:
I need to see in person the differences between what you have send and what I have in there.